File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1963.scala
import java.util.Scanner
import java.lang.Math

object Main {
    def main(args: Array[String]) {
        
        val ler = new Scanner(System.in)
        
        var a = ler.nextDouble();
        var b = ler.nextDouble();
  
        System.out.printf("%.2f%%\n", (b*100)/a-100);
    }
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1963.java
import java.util.Scanner;
import java.lang.Math;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        
        double a, b;
        a = ler.nextDouble();
        b = ler.nextDouble();
        
        System.out.printf("%.2f%%\n", (b*100)/a-100);
    }
}